app/serializers/v2/moves_serializer.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module V2
+  class MovesSerializer
+    include JSONAPI::Serializer
+
+    INCLUDED_ATTRIBUTES = %i[
+      additional_information
+      cancellation_reason
+      cancellation_reason_comment
+      created_at
+      date
+      date_from
+      date_to
+      move_agreed
+      move_agreed_by
+      move_type
+      reference
+      rejection_reason
+      status
+      time_due
+      updated_at
+    ].freeze
+
+    INCLUDED_FIELDS = {
+      moves: INCLUDED_ATTRIBUTES +
+        %i[profile from_location to_location prison_transfer_reason supplier additional_information],
+      profiles: ::V2::ProfileSerializer::INCLUDED_ATTRIBUTES + %i[person],
```

Comment:
we've previously seen an issue with having nested includes also duplicated at the higher level, is there a risk here where that will cause similar issues if we introduce new filters?

Reply:
I don't understand this point?
my aim here is to restrict the serializer for the moves index endpoint to all attributes plus a few select relationships

Reply:
the included field is indeed smaller on the index endpoint, however previously we've seen issues where specific filters might cause larger resources to be pulled in thus introducing some performance issues. I think since you've had a look/rejigged the ready in transit filter, and all the other filters have been tested on the FE we're ok.